### DISCUSSION TITLE
Is it possible to increase the menu font size in the Windows version of Geany 2.0 WITHOUT recompiling the program?

### DISCUSSION THREAD

**Discussion by dlrourke** [geany/geany#3653] [Q&A] [answered]
This is more a question than a bug report.

First, thank you very, very much for the new (2.0) version of Geany!
As my eyes aren't what they used to be, is it possible to increase the menu font size in the Windows version of Geany WITHOUT recompiling it.  An additional two points of font size would be great.

Thanks.

**Comment by b4n** [comment, marked as answer]
Yes, you can adjust this with custom [GTK CSS](https://docs.gtk.org/gtk3/css-overview.html).  Go to [*Tools → Configuration Files → geany.css*](https://geany.org/manual/#customizing-geany-s-appearance-using-gtk-css), and insert something like `menu { font-size: 15pt; }` or `* { -gtk-dpi: 120; }`, save and restart Geany.

You might be interested in [GTK CSS properties](https://docs.gtk.org/gtk3/css-properties.html) as well.

**Comment by dlrourke** [comment]
Dear b4n,
Thank you for your answer.  I ended up adding two changes to gtk.css:

menuitem { font-size: 13pt; }
menu { font-size: 12pt; }

Do these additions run the risk of breaking anything?  Right now, things look great.  So, for example "Tools" is now larger (13 points) and the sub-items under Tools are now at 12 points.  And things are much easier to read for my old eyes.

Thank you again!

**Comment by elextr** [comment]
@dlrourke your selected post said to edit "geany.css" not "gtk.css".  That way it only applies to Geany, not any other GTK application you may install.  

As for breaking anything, extremely unlikely, the most likely thing might be some text entries and labels not fitting in fixed spaces, but that won't "break" anything as in stop Geany from working, it might just be hard to read depending on how much is truncated.

**Reply by dlrourke** [reply]
I did edit 'geany.css' -- not 'gtk.css'.  A typo on my part.

On Windows, 12 pt and 13 pt fonts fit in the spaces for the menu item buttons without bumping into the edges.
